opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -172,9 +172,118 @@ message MetricDescriptor {
+    // form. If this state is lost or invalidated, the CUMULATIVE metric
+    // values MUST be reset and a new fixed start time following the last
+    // reported measurement time sent MUST be used.
+    //
+    // For example, consider a system measuring the number of requests that
+    // it receives every second and reports the sum of these requests as a
+    // CUMULATIVE metric:
+    //
+    //   1. The system starts receiving at time=t_0.
+    //   2. A request is received, the system measures 1 request.
+    //   3. A request is received, the system measures 1 request.
+    //   4. A request is received, the system measures 1 request.
+    //   5. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_0 to
+    //      t_0+1 with a value of 3.
+    //   6. A request is received, the system measures 1 request.
+    //   7. A request is received, the system measures 1 request.
+    //   8. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_0 to
+    //      t_0+2 with a value of 5.
+    //   9. The system experiences a fault and loses state.
+    //   10. The system recovers and resumes receiving at time=t_1.
+    //   11. A request is received, the system measures 1 request.
+    //   12. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_1 to
+    //      t_0+1 with a value of 1.
+    CUMULATIVE = 3;
+  }
+
+  // temporality is the Temporality of values this metric has.
+  Temporality temporality = 5;
+
+  // Monotonic is a refinement of the values a metric has. It defines the
+  // relationship values of successively reported metrics have
+  // (non-increasing, non-decreasing, or unknown). This is a refinement of
+  // the metric values that can be useful for a receiver in understanding
+  // how to deal with discontinuities in the data (i.e.  calculating
+  // derivates of the data without introducing artifacts from a reset).
+  enum Monotonic {
+    // MONOTONIC_UNSPECIFIED is the default, and means the monotonic nature
+    // of the metric values is unknown.
+    MONOTONIC_UNSPECIFIED = 0;
+
+    // NONDECREASING means all the successive metric values increase or
+    // remain constant.
+    NONDECREASING = 1;
+  }
+
+  // monotonic describes the Monotonic refinement of values this metric has.
+  Monotonic monotonic = 6;
+
+  // Domain is a refinement of the values a metric has. It describes the set
+  // of numbers metric values belong to, if any.
+  enum Domain {
+    // DOMAIN_UNSPECIFIED is the default, and means the metric values do
+    // not belong to any particular domain other than the Type itself.
+    DOMAIN_UNSPECIFIED = 0;
+
+    // NONNEGATIVE is the set of numbers greater than or equal to zero.
+    NONNEGATIVE = 1;
+  }
```

Comment:
Is this enough to replace monotonic?

Reply:
No. The concept of being a positive or zero number is decoupled from the concept of monotonicity.